### ISSUE TITLE
What should be the range for edit SFT loss value?

### ISSUE THREAD

**Issue by zhangquanwei962** [meituan-longcat/LongCat-Image#11]
<img width="506" height="378" alt="Image" src="https://github.com/user-attachments/assets/91d11d50-9193-4169-9020-97bbc7e60201" />

<img width="504" height="378" alt="Image" src="https://github.com/user-attachments/assets/94e3722c-21c2-4229-90cc-c339983b663c" />

<img width="490" height="335" alt="Image" src="https://github.com/user-attachments/assets/44c25f7f-67c1-4782-9979-86b29f0d2c31" />
I used about 50,000+ data pairs to train text editing, because I found that for instructions like "Replace 'xx1' to 'yy1'. Replace 'xx2' to 'yy2'", when each instruction needs to modify multiple text locations, it's not effective enough - it often only modifies one location. I plan to see the results after SFT, but I found that the training seems to have gone off track. Do you think this loss curve shows learning convergence around 1 epoch? Or would it be better if I use "Replace 'xx1 xx2' to 'yy1 yy2'" format instead?

**Comment by yayonguan** [comment]
Based on your style experimental results, the training code should be working. It is likely that the distribution or difficulty of your text-editing data differs from ours when we trained for text editing. You might try adjusting other hyperparameters, such as the learning rate, and observe whether this leads to any improvements.
